dataland-backend/src/main/kotlin/org/dataland/datalandbackend/service/DataManager.kt

```diff
@@ -0,0 +1,156 @@
+package org.dataland.datalandbackend.service
+
+import org.dataland.datalandbackend.edcClient.api.DefaultApi
+import org.dataland.datalandbackend.interfaces.DataManagerInterface
+import org.dataland.datalandbackend.model.CompanyMetaInformation
+import org.dataland.datalandbackend.model.DataManagerInputToGetData
+import org.dataland.datalandbackend.model.DataMetaInformation
+import org.dataland.datalandbackend.model.StorableDataSet
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+/**
+ * Implementation of a data manager for Dataland including meta data storages
+ */
+@Component("DefaultManager")
+class DataManager(
+    @Autowired var edcClient: DefaultApi
+) : DataManagerInterface {
+    var dataMetaData = mutableMapOf<String, DataMetaInformation>()
```

Comment:
metaData reicht auchals name, oder?

Reply:
Hier ist ja die Map data_id -> dataMetaInformation, oder?

Vielleicht wäre "metaInformationPerDataId" dann ein sprechenderer Name? Dito "CompanyMetaInformationPerCompanyId"

Reply:
Ich wollte damit explizit ausdrücken, dass diese metaData sich auf Data beziehen, und ausdrücklich nicht auf Companies oder etwas anderes.

Aber für mich wäre metaData auch fine. Was meinst du?

Reply:
wie besprochen `dataMetaInformationPerDataId`

Reply:
Fixed.  Habe aus Konsistenzgründen auch die Map `companyData` umbenannt in `companyDataPerCompanyId`. Passt?

Reply:
ja - danke!
passt.